src/main/external-resources/renderers/Panasonic-VieraST60.conf

```diff
@@ -12,11 +12,18 @@ RendererIcon = panasonic.png
 # User-Agent: Panasonic MIL DLNA CP UPnP/1.0 DLNADOC/1.50
 # X-PANASONIC-Registration: VklFUkEgU1Q2MCBTZXJpZXMAAAAAAAAAAAA=
 # friendlyName=VIERA ST60 Series
+#
+# # X-PANASONIC-DMP-Profile=""
+# X-PANASONIC-Registration: VklFUkEgU1Q2MAAAAAAAAAAAAAAAAAAAAAA=
```

Comment:
It's not a unique code.
Every informations sended, as `X-PANASONIC-DMP-Profile=""`, could be pertinent for later differentiation use, so writing it in the renderer is safe. Not every one can remember everything and running after log.